.github/workflows/mirror.yml

```diff
@@ -1,19 +1,19 @@
 # Mirror master to main branches in the cocoon repository.
-
 on:
   push:
     branches:
       - 'master'
 
 jobs:
   mirror_job:
+    permissions:
+      pull-requests: write
     runs-on: ubuntu-latest
     name: Mirror master branch to main branch
     steps:
-    - name: Mirror action step
-      id: mirror
-      uses: google/mirror-branch-action@c6b07e441a7ffc5ae15860c1d0a8107a3a151db8
-      with:
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        source: 'master'
-        dest: 'main'
```

Comment:
Is `dest` no longer needed here?

Reply:
Nice catch! it was removed as part of the conflict resolution. Added it back.